core/src/main/java/org/apache/calcite/sql/validate/SqlValidatorImpl.java

```diff
@@ -4901,7 +4902,8 @@ public void validateMerge(SqlMerge call) {
     if (call.getUpdateCall() != null) {
       validateUpdate(call.getUpdateCall());
     }
-    if (call.getInsertCall() != null && config.sqlConformance().allowMergeInsertRewrite()) {
+    if (call.getInsertCall() != null &&
```

Comment:
It looks like the checkstyle test fails unless "&&" is moved to the next line.

Reply:
Fix in https://github.com/googleinterns/calcite/pull/139/commits/eac9ee75ca51ce9dbc1efd141b1bfbe8ecb98e1a

Reply:
Somehow the Github action fails due to the "&&" operator even though changes are made.

Reply:
It's odd that the GitHub checks are still failing. When I run the autostyle checks locally on your branch, they pass. I've re-triggered the GitHub checks. Hopefully, they'll pass this time around.

Reply:
Re-run all the jobs does not solve the issue. I believe it is some Github action bug. Pushing another commit might help refreshing the cached source code in Github and end up passing the test. I might try that if this run still fails.

Reply:
It looks like it still fails. Digging into the GitHub test logs, it's definitely still referring to an older version of your code prior to moving the "&&" to a new line.

https://pipelines.actions.githubusercontent.com/xb7m9AKJAufNSYxCgYlBFp6zEZTsNRmGjs0J0ZIVck5oyJLlrb/_apis/pipelines/1/runs/1080/signedlogcontent/6?urlExpires=2020-07-07T16%3A24%3A18.1108720Z&urlSigningMethod=HMACV1&urlSignature=J6TJuli5w8S%2F7Nsv7%2F%2F2GvEQWPJwVMZXZCOTl6TuVTs%3D

Reply:
The logs are specifically calling that the tests are running on a commit from 7/6 @ 18:05:

2020-07-07T16:13:55.6345088Z [command]/usr/bin/git log -1
2020-07-07T16:13:55.6383774Z commit 83261a192e8ecf5432f95e115b88051bad07d152
2020-07-07T16:13:55.6427770Z Merge: 368ba6f 02a3323
2020-07-07T16:13:55.6427944Z Author: Junhao Chen <32620957+ArthurInTheShell@users.noreply.github.com>
2020-07-07T16:13:55.6428466Z Date:   Mon Jul 6 18:05:21 2020 -0700